pkg/pub_package_reader/lib/src/known_templates.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:pub_package_reader/pub_package_reader.dart';
+
+const _knownTemplateDescriptions = <String>{
+  // ex-stagehand templates, check latest versions in dart-lang/sdk's
+  // pkg/dartdev/lib/src/templates/ directory:
+  'a sample command-line application',
+  'a simple command-line application',
+  'a starting point for dart libraries or applications',
+  'a web server built using the shelf package',
+  'a web app that uses angulardart components',
+  'an absolute bare-bones web app',
+  'a simple stagexl web app',
+  // Flutter templates, check latest version in flutter/flutter's
+  // packages/flutter_tools/lib/src/commands/create.dart directory:
+  'a new flutter project',
+  'a new flutter module project',
+  'a new flutter package project',
+  'a new flutter plugin project',
+  'a new flutter ffi plugin project',
+};
+
+const _knownTemplateReadmes = <String>{
+  'todo: put a short description of the package here',
+  'todo: list what your package can do',
+  'todo: list prerequisites and provide or point to information on how to start using the package',
+  'todo: include short and useful examples for package users',
+  'todo: tell users more about the package',
+};
```

Comment:
How about we don't make it lower case? That'll produce a better error message. Or we do it on-the-fly when matching.